index.html

```diff
@@ -166,22 +165,23 @@
+The <dfn>`downlinkMax`</dfn> attribute represents the <dfn>upper bound on the downlink speed of the first network hop</dfn>. The reported value is in <a>megabits per second</a> and determined by the properties of the <a>underlying connection technology</a>.
 
-</section>
-
-<section>
+<div class="note">
+The user agent has no knowledge of the total number or the performance characteristics of the various network hops required to fulfill a particular request; different requests may follow different routes and have different performance characteristics. The reported <a>upper bound on the downlink speed of the first network hop</a> value is determined by the properties of the <a>underlying connection technology</a> of the first network hop. The end-to-end performance of any request cannot exceed this value, but it is also not a guarantee of performance and may be significantly worse.
+</div>
 
 ## Underlying connection technology
-The <dfn>underlying connection technology</dfn> represents the generation and/or version of the network connection being used by the device. For example, "HSPA" (3.5G) for cellular, or "802.11g" for Wi-Fi. This differs from the <a title="connection types">connection type</a> exposed by the API, in that it is more fine grained.
 
-Each <a>underlying connection technology</a> has an associated <dfn>maximum downlink speed</dfn>, which is the maximum download data rate of the current network interface in <a>megabits per second</a>. The relationship between an <a>underlying connection technology</a> and its <a>maximum downlink speed</a> is determined by the properties of the current network interface (e.g signal strength, modulation algorithm, and other relevant "network weather" variables), or where such interface information is not available, is determined by the standardized, or generally accepted, maximum download data rate captured in the <a>table of maximum downlink speeds</a>.
+The relationship between an <dfn>underlying connection technology</dfn> and its <a>upper bound on the downlink speed of the first network hop</a> is determined by the current properties of the primary network interface that will be used to fulfill new network requests: type and version of the interface and relevant "network weather" variables that may affect its performance - e.g. signal strength, modulation algorithm, and so on.
+
+In absence of relevant "network weather" information from the primary interface the <a>upper bound on the downlink speed of the first network hop</a> is determined via the standardized, or generally accepted, maximum download data rate captured in the <a>table of maximum downlink speeds</a>. When additional "network weather" information is available, it may be used to report a more accurate <a>upper bound on the downlink speed of the first network hop</a> - e.g. lower rate due to poor signal quality.
 
```

Comment:
Regarding "primary network interface" - I'm not sure how that's defined and if we should limit ourselves to that.

If we want to tackle the multipath scenario, maybe we should define `downlinkMax` as the upper bound of all active interfaces.